docker-compose.yml

```diff
@@ -408,7 +408,7 @@ services:
       - INFINISPAN_URL=infinispan:${INFINISPAN_PORT}
       - INFINISPAN_CLIENT_INTELLIGENCE=BASIC
       - GENNY_KAFKA_URL=kafka:${KAFKA_PORT}
-      - GENNY_KEYCLOAK_URL=https://keycloak-bali.gada.io
```

Comment:
Since the variable name remains the same, the entire line could be removed.

Reply:
done